backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to {oadp-full}
+include::_attributes/common-attributes.adoc[]
```

Comment:
In order to have the `{oadp-full}` variable render in the heading on the Customer Portal, you need to move the `include::` statement above the heading. You can see an example in the [contributor guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata).

This quirk of the Customer Portal is something we only found out about recently. It was brought up at the OpenShift docs weekly team meeting today, 21 August 2023.
```suggestion
[id="oadp-introduction"]
include::_attributes/common-attributes.adoc[]
= Introduction to {oadp-full}
```